service-commands/scripts/A.py

```diff
@@ -0,0 +1,243 @@
+import subprocess
+import argparse
+
+
+def create_instance(name, image_project, image_family, size, machine_type):
+    subprocess.run(
+        [
+            "gcloud",
```

Comment:
We should do some gcloud settings validation like see if you're in the right account and project

Reply:
Also what about firewall? They need to have that already configured right? Or does that not matter?